docs/Configuration.md

```diff
@@ -1,206 +1,206 @@
-# Configuration File
+# Configuration
 
-## Overview
+Cayley can be configured using configuration file written in YAML / JSON or by passing flags to the `http` command line.
+By default. All command line flags take precedence over the configuration file.
 
-Cayley expects, in the usual case, to be run with a configuration file, though it can also be run purely through configuration flags. The configuration file contains a YAML/JSON object with any of the documented parameters.
+- [Recommended Configuration](#Recommended-Configuration)
+- [Configuration Options](#Configuration-Options)
+  - [Store](#Store)
+  - [Per-Store Options](#Per-Store-Options)
+  - [Query](#Query)
+  - [Load](#Load)
+- [Configuration File Location](#Configuration-File-Location)
 
-Cayley looks in the following locations for the configuration file (named `cayley.yml` or `cayley.json`):
+## Recommended Configuration
 
-  * Command line flag
-  * The environment variable $CAYLEY_CFG
-  * Current directory.
-  * $HOME/.cayley/
-  * /etc/
+By default, Cayley is using an on-disk `bolt` store. This will best work for most use cases.
```

Comment:
No, I think by default it uses `memstore`, but `bolt` is indeed recommended for a basic persistent configuration.

Reply:
I think the Docker is inconsistent with the CLI. Should we fix the Docker, or change the CLI?

Reply:
I opened a different PR to fix the Dockerfile: https://github.com/cayleygraph/cayley/pull/842